### PR TITLE
List formatting

### PR DESCRIPTION
**Pull request by dburka**
Lots of formatting:

- Fixed several broken icons
- Formatted the icons lists in the grid
- Made the link color purple
- Added an About page
- Added a Footer to the homepage

### CODE REVIEW

File: src/pages/about.tsx [hunk 1]
```diff
@@ -5,13 +5,106 @@ export default function About() {
...
+          <div className="contentMain">
+            <h1 className="title">About</h1>
+            <p>Health Icons is a volunteer effort to create a 'global good' of useful icons for health projects all over the world. We intend that these icons are available in the public domain for use in any type of project.</p>
+            <p>Need an icon for your project? Submit a request through Github and we will do our best to respond.</p>
+
+            <h2>How to contribute</h2>
+            <p>Join our open Slack channel by emailing <a href="mailto:contact@healthicons.org">contact@healthicons.org</a></p>
+            <p>Browse the <a href="https://www.figma.com/file/mbsBVYXECIOl5E0kkGAiC2/?node-id=978%3A3">Figma</a> and contribute icons in the "New icons" section. Please read the code of conduct (below) and the styleguide (in the Figma) carefully before contributing.</p>
+
+            <h2 id="license">License</h2>
```
Comment (nburka) [inline src/pages/about.tsx:25]: i think this should use `name`

Comment (nburka) [reply src/pages/about.tsx:25]: also, if we add an anchor here, we could add it for the other H2's on this page

**Review by nburka — CHANGES_REQUESTED** [review]

File: src/components/TopBar.module.scss [hunk 2]
```diff
@@ -3,8 +3,11 @@
   display: flex;
 }
 
-.title {
+.title, .title:visited {
   font-weight: bold;
+  font-size: 1.3rem;
+  margin-top: -0.3rem;
+  text-decoration: none !important;
```
[the remark this answers is not in the document]
Comment (dburka) [reply src/components/TopBar.module.scss:10]: No, but I can't figure it out :/

File: src/components/TopBar.module.scss [hunk 3]
```diff
@@ -13,5 +16,13 @@
 
 .link {
   display: inline-block;
-  padding: 0 0 0 2rem;
+  margin: 0 0 0 2rem;
+}
+
+.buttonLink {
+  background: var(--link-color);
+  color: #fff !important;
```
[the remark this answers is not in the document]
Comment (dburka) [reply src/components/TopBar.module.scss:24]: No, but I can't figure it out :/

[the remark this answers is not in the document]
Comment (nburka) [reply src/components/TopBar.module.scss:24]: I made an Issue to fix this later

File: src/components/TopBar.module.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (nburka) [reply src/components/TopBar.module.scss:10]: I made an Issue to fix this later

**Pull request merged**